### PR TITLE
feat(daypicker): Add markup and styles to display price information

### PR DESCRIPTION
**Pull request by rachmulvey**
This change is [Reviewable](https://reviewable.io/reviews/qantasairways/runway/49)

### CODE REVIEW

File: src/components/DayPicker/components/Day.js [hunk 1]
```diff
@@ -134,15 +139,18 @@ function getAriaLabel({
 
 class Day extends Component {
   state = {
-    hover: false
+    hover: false,
+    isLoadingPrice: true
```
Comment (Pallandor) [inline src/components/DayPicker/components/Day.js]: I can't find where the state tracked `isLoadingPrice` is being updated. Is it still required?

Comment (rachmulvey) [reply src/components/DayPicker/components/Day.js]: Yeah nice catch, this was just for testing

File: src/components/DayPicker/components/Day.js [hunk 2]
```diff
@@ -140,9 +145,11 @@ class Day extends Component {
   shouldComponentUpdate(nextProps, nextState) {
     if (
       nextState.hover !== this.state.hover ||
+      nextState.isLoadingPrice !== this.state.isLoadingPrice ||
```
Comment (Pallandor) [inline src/components/DayPicker/components/Day.js]: Should probably strip out this check too `nextState.isLoadingPrice !== this.state.isLoadingPrice`

**Pull request merged**